azure_pipelines/pr-validation.yml

```diff
@@ -0,0 +1,73 @@
+# Pipeline will be triggered for PR & any updates on the PR on all branches
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->